pkg/heartbeat/sanitize.go

```diff
@@ -64,6 +67,22 @@ func Sanitize(h Heartbeat, config SanitizeConfig) Heartbeat {
 		h.Branch = nil
 	}
 
+	if h.EntityType == FileType && config.HideProjectFolder {
+		// normalize project path to look like a directory
+		if h.ProjectPath != "" && !strings.HasSuffix(h.ProjectPath, "/") {
+			h.ProjectPath += "/"
```

Comment:
Let's also replace `\` with `/` in `h.ProjectPath` using `windows.FormatFilePath`.

Reply:
Formatting auto-detected file path into project detection where it's already set.